### PR TITLE
chore(es): handling redirects of chrome.com

### PR DESCRIPTION
**Pull request by PassionPenguin**
1. handling redircts of `chrome.com`

### CODE REVIEW

**Comment by PassionPenguin** [comment]
> [!WARNING]
> Entries related to chrome devtools may need a thorough cleanup. Links pointing to chrome devtools have a tremendous change and such documents on google developer site are no longer available.

File: files/es/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md [hunk 1]
```diff
@@ -227,7 +227,7 @@ Obtén más información sobre la consola de JavaScript en diferentes navegadore
 
 - [Consola Web de Firefox.](/es/docs/Tools/Web_Console)
 - [Consola de JavaScript Edge.](https://docs.microsoft.com/es-es/microsoft-edge/devtools-guide-chromium)
-- [Consola JavaScript de Chrome.](https://developer.chrome.com/devtools/docs/console) (el inspector de Opera funciona de la misma manera)
+- [Consola JavaScript de Chrome.](https://developers.google.com/web/tools/chrome-devtools/?hl=es-419) (el inspector de Opera funciona de la misma manera)
```
Comment (PassionPenguin) [inline files/es/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md:230]:
```suggestion
- [Consola JavaScript de Chrome.](https://developer.chrome.com/docs/devtools/?hl=es-419) (el inspector de Opera funciona de la misma manera)
```

File: files/es/web/api/console/assert_static/index.md [hunk 2]
```diff
@@ -46,4 +46,4 @@ Para más detalles ver [Salida de texto a la consola](/es/docs/Web/API/console#O
 
 - [Opera Dragonfly documentación: Consola](http://www.opera.com/dragonfly/documentation/console/)
 - [MSDN: Usando F12 Herramientas de Consola para ver Errores y Estados](http://msdn.microsoft.com/library/gg589530)
-- [Herramientas de Desarrollo de Chrome: Usando la Consola](https://developer.chrome.com/devtools/docs/console#assertions)
+- [Herramientas de Desarrollo de Chrome: Usando la Consola](https://developers.google.com/web/tools/chrome-devtools/?hl=es-419#assertions)
```
Comment (PassionPenguin) [inline files/es/web/api/console/assert_static/index.md:49]:
```suggestion
- [Herramientas de Desarrollo de Chrome: Usando la Consola](https://developer.chrome.com/docs/devtools/?hl=es-419#assertions)
```

File: files/es/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md [hunk 3]
```diff
@@ -227,7 +227,7 @@ Obtén más información sobre la consola de JavaScript en diferentes navegadore
 
 - [Consola Web de Firefox.](/es/docs/Tools/Web_Console)
 - [Consola de JavaScript Edge.](https://docs.microsoft.com/es-es/microsoft-edge/devtools-guide-chromium)
-- [Consola JavaScript de Chrome.](https://developer.chrome.com/devtools/docs/console) (el inspector de Opera funciona de la misma manera)
+- [Consola JavaScript de Chrome.](https://developer.chrome.com/docs/devtools/) (el inspector de Opera funciona de la misma manera)
```
Comment (Graywolf9) [inline files/es/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md:230]:
```suggestion
- [Consola JavaScript de Chrome.](https://developer.chrome.com/docs/devtools/console/) (el inspector de Opera funciona de la misma manera)
```

File: files/es/web/api/console/assert_static/index.md [hunk 4]
```diff
@@ -46,4 +46,4 @@ Para más detalles ver [Salida de texto a la consola](/es/docs/Web/API/console#O
 
 - [Opera Dragonfly documentación: Consola](http://www.opera.com/dragonfly/documentation/console/)
 - [MSDN: Usando F12 Herramientas de Consola para ver Errores y Estados](http://msdn.microsoft.com/library/gg589530)
-- [Herramientas de Desarrollo de Chrome: Usando la Consola](https://developer.chrome.com/devtools/docs/console#assertions)
+- [Herramientas de Desarrollo de Chrome: Usando la Consola](https://developer.chrome.com/docs/devtools/#assertions)
```
Comment (Graywolf9) [inline files/es/web/api/console/assert_static/index.md:49]:
```suggestion
- [Herramientas de Desarrollo de Chrome: Usando la Consola](https://developer.chrome.com/docs/devtools/console/)
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for your time and help!

**Pull request merged**